tests/pre_post_condition/include/src.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+
+void f1(int a, int *b);
```

Comment:
We should have a few tests with a different number of arguments. They don't have to be extensive, but in particular I think a function with an odd number of arguments that calls `printf` (which often uses x86 simd instructions that assume rsp is aligned to 16 bytes before every function call) is likely fail because this PR currently misaligns the stack.